controllers/feeds/getFeedChat.js

```diff
@@ -1,6 +1,7 @@
 const moment = require('moment');
+const {Sequelize} = require('sequelize');
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `Sequelize` from the `sequelize` package is unnecessary because you are not using it directly. Instead, you are using an instance of Sequelize (`sequelize`) which is imported from your models. This instance already has all the necessary methods and properties.
```diff
- const {Sequelize} = require('sequelize');
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->